doc/governance.rst

```diff
@@ -1,3 +1,4 @@
+:orphan:
```

Comment:
I think these should rather be added to our Sphinx TOC. Thanks for catching them!!

Reply:
At least the `roadmap` is already linked from the main page already. I'll remove these for now and someone can link in a follow-up PR so that this can go in. Numpydoc 0.9.0 was just released so the other stuff below will be problematic